server/startup/migrations/v069.js

```diff
@@ -0,0 +1,14 @@
+RocketChat.Migrations.add({
+	version: 69,
+	up: function() {
+		const settings = RocketChat.models.Settings.find({ _id: /^Accounts_OAuth_Custom_.+/ }).fetch();
```

Comment:
@graywolf336 That rule is from JSHint, I will disable all JSHint items and enable only ESLint items :)